user_guide_src/source/libraries/pagination/012.php

```diff
@@ -1,3 +1,39 @@
-<?php
+<?php $pager->setSurroundCount(2) ?>
 
-'default_full'   => 'Pagers/foundation_full',
+<nav aria-label="Page navigation">
+    <ul class="pagination">
+    <?php if ($pager->hasPrevious()) : ?>
+        <li>
+            <a href="<?= $pager->getFirst() ?>" aria-label="<?= lang('Pager.first') ?>">
+                <span aria-hidden="true"><?= lang('Pager.first') ?></span>
+            </a>
+        </li>
+        <li>
+            <a href="<?= $pager->getPrevious() ?>" aria-label="<?= lang('Pager.previous') ?>">
+                <span aria-hidden="true"><?= lang('Pager.previous') ?></span>
+            </a>
+        </li>
+    <?php endif ?>
+
+    <?php foreach ($pager->links() as $link): ?>
+        <li <?= $link['active'] ? 'class="active"' : '' ?>>
+            <a href="<?= $link['uri'] ?>">
+                <?= $link['title'] ?>
+            </a>
+        </li>
+    <?php endforeach ?>
+
+    <?php if ($pager->hasNext()) : ?>
+        <li>
+            <a href="<?= $pager->getNext() ?>" aria-label="<?= lang('Pager.next') ?>">
+                <span aria-hidden="true"><?= lang('Pager.next') ?></span>
+            </a>
+        </li>
+        <li>
+            <a href="<?= $pager->getLast() ?>" aria-label="<?= lang('Pager.last') ?>">
+                <span aria-hidden="true"><?= lang('Pager.last') ?></span>
+            </a>
+        </li>
+    <?php endif ?>
+    </ul>
+</nav>
```

Comment:
This change is because of renumbering of the source files,
but generally it is difficult to review.

Probably we should not mix the content changes and source file renumbering.

Reply:
This might make sense.
You can also just click on the commit-id on the right to only see changes by this one commit. This way you can avoid seeing the renumeration for now.